iocage/lib/Config/Jail/Properties/Interfaces.py

```diff
@@ -24,36 +24,7 @@
 import typing
 
 import iocage.lib.helpers
-
-
-class BridgeSet(set):
```

Comment:
i would love to read a bit more (than just "sinularize vnet bridges") of an explanation for removing this whole class

it might also aid a lot in understanding the rest of the code(:

Reply:
This is a bit awkward. In the first place a device could have multiple bridges it gets assigned to - that does not make sense at all. We never used multiple bridges, but used BridgeSet to store a single item.